tests/automation/utilities/utils.ts

```diff
@@ -499,6 +499,9 @@ export async function checkModalStrings(
   expectedHeading: string,
   expectedDescription: string,
 ) {
+  function removeNewLines(input: string): string {
+    return input.replace(/\s+/g, ' ').trim();
+  }
```

Comment:
should probably be a function in a utils file, or at least outside of the test itself.,